src/NHibernate/Mapping/ByCode/IColumnsAndFormulasMapper.cs

```diff
@@ -0,0 +1,198 @@
+using System;
+using NHibernate.Util;
+
+namespace NHibernate.Mapping.ByCode
+{
+	/// <summary>
+	/// A mapper for mapping mixed list of columns and formulas.
+	/// </summary>
+	public interface IColumnsAndFormulasMapper : IColumnsMapper
+	{
+		/// <summary>
+		/// Maps a mixed list of columns and formulas.
+		/// </summary>
+		/// <param name="columnOrFormulaMapper">The mappers for each column or formula.</param>
+		/// <remarks>Replaces any previously mapped column or formula.</remarks>
+		void ColumnsAndFormulas(params Action<IColumnOrFormulaMapper>[] columnOrFormulaMapper);
```

Comment:
I still do not have a better name. In fact, I do not think there is a better name fitting the current API shape, which already does not follow usual guidelines for method names. (For example, they are not prefixed by an imperative verb.)